### ISSUE TITLE
change label of tumor terms to be neoplasm

### ISSUE THREAD

**Issue by obophenotype-user** [obophenotype/upheno#103]
Originally reported on Google Code with ID 102
```
tumoriogenesis branch of MP

reconcile high level term labels with neoplasm vs. tumor

neoplasm convers all types of neoplastic change- tumors are defined as a subclass of
these and are pathologically defined as solid types of aequal to or greater than a
certain size, so the term 'tumor' excludes leukemias for example.




```

Reported by `cynthia.smith@jax.org` on 2015-02-13 00:34:31

**Comment by drseb** [comment]
not sure we should move this to MP tracker!?

**Comment by cindyJax** [comment]
Moved to MP tracker
